### ISSUE TITLE
First WebApp / Front End

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#20]
## User story
1. As a user
2. I want / need a WebApp I can open
3. So that I can put in text

## Acceptance criteria
* I can open the Page
* I can input text
* The text input is reaching the backend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
